src/main/java/se/kth/sda/skeleton/user/User.java

```diff
@@ -30,16 +32,17 @@
     private String email;
 
 
-    @Length(min = 5, max=100, message = "Password length most be between 5-100 characters")
+    @Length(min = 5, max = 100, message = "Password length most be between 5-100 characters")
     @Column(name = "password")
     private String password;
 
-    @Length(min = 3, max=100, message = "Name must be between 3-100 characters")
+    @Length(min = 3, max = 100, message = "Name must be between 3-100 characters")
     @Column(name = "name")
     private String name;
 
     // Hibernate needs a default constructor to function
-    public User() {}
+    public User() {
+    }
 
     public User(@Email(message = "Invalid email address! Please provide a valid email address") @NotEmpty(message = "Please provide an email address") String email, @Length(min = 5, max = 100, message = "Password length most be between 5-100 characters") String password, @Length(min = 3, max = 100, message = "Name must be between 3-100 characters") String name) {
```

Comment:
would min updating the message for password? It has a typo saying "most" when it should be "must"

Reply:
Changed